### PR TITLE
WSTEAM1-288: Add Twitter to consent list

### PR DESCRIPTION
**Pull request by shayneahchoon**
Resolves #NUMBER

**Overall change:**
_A very high-level summary of easily-reproducible changes that can be understood by non-devs._

**Code changes:**

- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/legacy/containers/SocialEmbed/Cps/index.stories.jsx [hunk 1]
```diff
@@ -23,27 +24,33 @@ export default {
 };
 
 export const CanonicalExample = props => (
-  <Component
-    blocks={[cpsTwitterBlock]}
-    source="https://twitter.com/MileyCyrus/status/1237210910835392512"
-    {...props}
-  />
+  <ThemeProvider service="news">
```
Comment (amoore108) [inline src/app/legacy/containers/SocialEmbed/Cps/index.stories.jsx]: May be easier to add this to the `withContexts` helper function

File: src/app/legacy/containers/SocialEmbed/index.test.jsx [hunk 2]
```diff
@@ -39,17 +39,8 @@ describe('SocialEmbedContainer', () => {
...
-          'head script[src="https://platform.twitter.com/widgets.js"]',
-        ),
-      ).toBeTruthy();
+      
       unmount();
-      expect(
-        document.querySelector(
-          'head script[src="https://platform.twitter.com/widgets.js"]',
-        ),
-      ).toBeFalsy();
```
Comment (amoore108) [inline src/app/legacy/containers/SocialEmbed/index.test.jsx]: Do these need removed? Looks like if the fireEvent is added it should trigger rendering of the embed: https://github.com/bbc/simorgh/commit/fb200acefdc1feaee78f34ae0605536ed56c52c1#diff-ff444dffb733dcf5993367397d172904ce047824c159d8c6449882e305fe29b1

File: src/app/legacy/psammead/psammead-social-embed/src/index.test.jsx [hunk 3]
```diff
@@ -128,19 +128,12 @@ describe('CanonicalSocialEmbed', () => {
...
-        document.querySelector(
-          'head script[src="https://platform.twitter.com/widgets.js"]',
-        ),
-      ).toBeTruthy();
-      unmount();
-      expect(
-        document.querySelector(
-          'head script[src="https://platform.twitter.com/widgets.js"]',
-        ),
-      ).toBeFalsy();
```
Comment (amoore108) [inline src/app/legacy/psammead/psammead-social-embed/src/index.test.jsx]: Similar to the above: https://github.com/bbc/simorgh/pull/10562/commits/fb200acefdc1feaee78f34ae0605536ed56c52c1#diff-7432987f0acea427fd1311feb48d68d7f5aa675f3bb25f45c4e34db0d1e6275b

**Comment by amoore108** [comment]
Just a couple of q's, but if its too time consuming to fix then I'm happy to fix forward 👍🏼 

Just some linting errors.

File: src/app/legacy/containers/SocialEmbed/index.test.jsx [hunk 4]
```diff
@@ -39,17 +39,8 @@ describe('SocialEmbedContainer', () => {
       );
 
       expect(container.firstChild).toMatchSnapshot();
-      expect(
-        document.querySelector(
-          'head script[src="https://platform.twitter.com/widgets.js"]',
-        ),
-      ).toBeTruthy();
+
```
Comment (HarveyPeachey) [inline src/app/legacy/containers/SocialEmbed/index.test.jsx]: This deleted test here you can import fireEvent from `react-testing-library-with-providers` and do the same click invoking and it should work

**Review by HarveyPeachey — APPROVED** [review]
Just a suggestion on a deleted test

**Review by amoore108 — APPROVED** [review]

**Pull request merged**